docs/examples/fd_fitting_basic/index.rst

```diff
@@ -0,0 +1,194 @@
+
+And fit the model::
+
+    >>> fit_odijk.fit()
+
+    Fit
+      - Model: DNA(x-d)_with_DNA
+      - Equation:
+          f(d) = argmin[f](norm(DNA.Lc * (1 - (1/2)*sqrt(kT/(f*DNA.Lp)) + f/DNA.St)-(d - DNA.d_offset))) + DNA.f_offset
+
+      - Data sets:
+        - FitData(Inverted Odijk, N=959)
+
+      - Fitted parameters:
+        Name                  Value  Unit      Fitted      Lower bound    Upper bound
+        ------------  -------------  --------  --------  -------------  -------------
+        DNA/d_offset    -0.00601252  [au]      True              -0.01           0.01
+        DNA/Lp          44.2558      [nm]      True               0            100
+        DNA/Lc           2.80085     [micron]  True               0            inf
+        DNA/St        1714.56        [pN]      True               0            inf
+        kT               4.11        [pN*nm]   False              0              8
+        DNA/f_offset     0.0392458   [pN]      True              -0.1            0.1
+
+Set up the Twistable worm like chain model
+------------------------------------------
+
+Set up a twistable worm like chain model with a distance and force offset. By default,
+the `twistable_wlc` model provided with pylake is defined as distance as a function of
+force. Typically, we want to fit force as a function of distance however. To achieve
+this, we can invert the model using its `invert` function. However, this can be slow
+in certain cases, as it requires an inversion for each data point. Luckily we have a
+faster way of achieving this in pylake, which is to use the dedicated `inverted_twistable_wlc`
+model.
+
+Again, we incorporate an offset in both distance and force to compensate for small
+offsets that may exist in the data:::
+
+    m_dna = pylake.inverted_twistable_wlc('DNA').subtract_independent_offset() + pylake.force_offset('DNA')
+    fit_twlc = pylake.FdFit(m_dna)
+
+Load the data into the model
+----------------------------
+
+Now it is time to load the full data into the model. Note in the figure however,
+that there seems to be a small break at the end of the Fd curve. The model will
+not be able to capture this behaviour, and therefore it is best to remove it
+prior to fitting as it will otherwise bias our parameter estimates::
+
+    force = fd_curve.f.data
+    distance = fd_curve.d.data
+    mask = distance < 2.88
+    distance = distance[mask]
+    force = force[mask]
+
+Now we can load the data into the model::
+
+    fit_twlc.add_data("Twistable WLC", force, distance)
+
+We could add more datasets in a similar manner, but in this example, we only fit a single model. Let’s load the
+parameters from our previous fit to use them as initial guesses for this one. We also fix the twist rigidity and
+critical force values to values from literature (analogous to Broekmans et al. "DNA twist stability changes with
+magnesium (2+) concentration." Physical review letters 116.25 (2016))::
+
+    fit_twlc << fit_odijk
```

Comment:
I guess that this means "load parameters from our previous fit" but it's far removed from the text. I would strongly recommend making this a named method instead of the `<<` operator. I'm very much for operator overloading in cases of frequently used constructs. It can make things easier to read, especially for chaining. But here I don't see an obvious advantage, especially since I see it being used only once in all the examples. It takes more time to explain than to use if it's used very rarely.

Reply:
Yeah, that's fair. How about `load_parameters_from`?

`load_parameters` makes me think it'll overwrite what parameters are in the list (the keys as well as the values) whereas this only assigns to keys that already exist in the target and doesn't require that all keys are even present.

Reply:
Maybe something closer to the intention? `.set_initial_guess()`? In any case, avoid the word `load` as mentioned before in the context of setting datasets.

Reply:
Hmm, I'm not sure. It has more uses than just initial guesses though. In some cases, you may want to load them but fix them, when you fix a DNA linker for instance. In that case it would hardly function as an initial guess. Initial guess also makes me think that it would have to give a complete parameter vector, while this is typically used to provide values for a subset.

I'd prefer something closer to what it does like `set_parameters`. The reason I preferred `load` over `set` was because `set` sounded a bit like it would override the complete vector (adding ones that weren't in the target for instance). `import_parameter_values` maybe? I don't know. I'll sleep on it! :D

Reply:
`.update_params()`? Update seems to be the thing that's happening.

* I also noticed that `parameters` vs `params` in the API. Would be good to unify everything. My vote is for `params` since it's shorter :)

Reply:
👍

Reply:
`params` it is. Also, `update_params()` sounds good. 👍

Reply:
`params` and `.update_params()` it is! :)